portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -456,6 +456,28 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	macro clearProperty {
+		ACUtils.searchBar(searchItem = "${propertyName}");
+
+		ACNavigation.openItem(itemName = "${propertyName}");
+
+		ACUtils.clickAnyButton(buttonName = "Clear Data");
+
+		Copy(locator1 = "ACSettings#DELETE_CONFIRMATION_MESSAGE");
+
+		Click(locator1 = "ACSettings#DELETE_CONFIRMATION_INPUT");
+
+		Paste(locator1 = "ACSettings#DELETE_CONFIRMATION_INPUT");
+
+		Click(locator1 = "ACSettings#DELETE_CONFIRMATION_BUTTON");
+
+		Alert.viewSuccessMessage();
+
+		// This pause is necessary as it takes time for the data to be cleared from the property. See LRAC-11595
+
+		Pause(locator1 = "10000");
```

Comment:
Hey @joshitagaki ! Thanks for working on this, the code LGTM. I have a question, since we want to document our pauses under `ACUtils.macro`. And maybe for future usage of data clearing pauses, do you think we should make a pause macro for clearing data?

Reply:
Wouldn't this be the macro to use to track the data clearing? Since the way to clear the data is through clearing the property

Reply:
Talked with @calvinctvo. Moving the pause to a separate macro located in the ACUtils macro under waitForClearProperty. Easier to track all pause macros that are located in a single file